src/pybind/mgr/dashboard/services/feedback.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+
+from typing import Optional
+from redminelib import Redmine
+from enum import Enum
+import errno
+
+from requests.auth import AuthBase
+from mgr_module import CLICommand, CLIWriteCommand, CLIReadCommand, HandleCommandResult
+
+from .. import mgr
+from ..settings import Settings
+from ..exceptions import DashboardException, ScopeNotInRole
+from ..rest_client import RestClient
+
+
+class Feedback(Enum):
+    project_id = 1,
+    tracker_id = 1,
+    subject = '',
+    description = '',
+    category_id = 195,
+    Severity = '3 - Minor'
```

Comment:
Enums are for related collections of constants:
```py
class Color(Enum):
  RED = 0xff0000
  GREEN = 0x00ff00
  BLUE = 0x0000ff
```

And then, instead of using `0xff0000` everywhere, you use `Color.RED`.

In this very case what you need is a Model aka Plain Object aka Data class:
```py
class Feedback:
  project_id: int
  tracker_id: int
...
  def __init__(self, project_id, tracker_id, ...):
    self.project_id = project_id
    ...

  @classmethod
  def from_db(cls, tracker_id): # in this very case the DB is the Redmine REST API
    """tracker = ... GET https://tracker.ceph.com/issue=tracker_id"""
    return cls(tracker.project_id, tracker_id, ...)

  def as_dict(self):
    return {...}

  def to_db(self):
    """POST https://tracker.ceph.com/... body = self.as_dict()"""
    
```

Reply:
Thank you for explaining. I'll make these changes.